test.js

```diff
@@ -43,3 +43,33 @@ test('successfully promisifies function with properties', () => {
     expect(promisified.bar()).resolves.toEqual('bar'),
   ]);
 });
+
+test('leave primitives alone', () => {
+  const number = 1;
+  const boolean = true;
+  const string = 'string';
+  const nullVar = null;
+  const undefinedVar = undefined;
+  const symbol = Symbol('some symbol');
+
+  expect(m(number)).toBe(number);
+  expect(m(boolean)).toBe(boolean);
+  expect(m(string)).toBe(string);
+  expect(m(nullVar)).toBe(nullVar);
+  expect(m(undefinedVar)).toBe(undefinedVar);
+  expect(m(symbol)).toBe(symbol);
+});
+
+test('promisify array properties', () => {
+  const array = [true, 'sup', func()];
+  array.bar = func();
+
+  expect(m(array)).not.toBe(array);
+  return expect(m(array).bar()).resolves.toBeDefined();
+});
+
+test('return same array if no promisification', () => {
+  const array = [true, 'sup', func()];
+
+  expect(m(array)).toBe(array);
```

Comment:
this should be `toEqual`; `toBe` is `===`.

Also, I think you'll have to compare `entries(m(array))` to `entries(array)`, since the test framework probably only considers indexed properties on arrays.